### PR TITLE
Changes related to debugging the scanner

### PR DESCRIPTION
**Pull request by bplaxco**

### CODE REVIEW

**Comment by bplaxco** [comment]
@thewizzy a WIP but at least this lets us bubble up logs from gitleaks now :+1:

File: pkg/scanner/gitleaks.go [hunk 1]
```diff
@@ -111,6 +112,7 @@ func (g *Gitleaks) gitScan(detector *detect.Detector, gitRepo *resource.GitRepo)
 		return nil, err
 	}
 
+	logger.Debug("running detector.DetectGit")
```
Comment (thewizzy) [inline pkg/scanner/gitleaks.go:115]: Do the logger.Debug()'s here need to stay around?

**Review by thewizzy — APPROVED** [review]
Approved pending gitleaks.go feedback.

File: pkg/scanner/gitleaks.go (hunk 1 above)
Comment (bplaxco) [reply pkg/scanner/gitleaks.go:115]: I'll go ahead and take it out for now. I'm on the fence about it being useful

**Comment by bplaxco** [comment]
> Approved pending gitleaks.go feedback.

Removed unnecessary logging statement.

**Pull request merged**